<ui version="4.0" >
 <class>NA18DeliveryNoteCopyVCClass</class>
 <widget class="QWidget" name="NA18DeliveryNoteCopyVCClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NA18 Delivery Note Copy</string>
  </property>
  <widget class="QGroupBox" name="grpRectificationNote" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>290</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title" >
    <string>Rectification note</string>
   </property>
   <widget class="QTextEdit" name="rectificationcomment" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>100</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lbRectificationComment" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Additional comment</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="rectificationtext" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFormatting" >
     <set>QTextEdit::AutoNone</set>
    </property>
    <property name="undoRedoEnabled" >
     <bool>false</bool>
    </property>
    <property name="acceptRichText" >
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags" >
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpRectificationReason" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>290</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Rectification reason</string>
   </property>
   <widget class="QRadioButton" name="rbTheftLoss" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text" >
     <string>Theft / Loss</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbRefusalOfAcceptance" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text" >
     <string>Customer refusal of acceptance</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbQuantityCorrection" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text" >
     <string>Quantity correction / Damaged</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK" >
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>470</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>470</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
   <property name="autoDefault" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpContributionVoucher" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title" >
    <string>Contribution voucher</string>
   </property>
   <widget class="QLabel" name="labelContributionVoucherNo" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Contribution voucher no</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCustomerNo" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Customer no</string>
    </property>
   </widget>
   <widget class="BasarTableWidget" name="tabPositions" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>90</y>
      <width>511</width>
      <height>171</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy" >
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy" >
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="selectionMode" >
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior" >
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount" >
     <number>6</number>
    </property>
    <property name="columnCount" >
     <number>6</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column>
     <property name="text" >
      <string>Article No</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Quantity</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Original Quantity</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Packing Unit</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Article Name</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelPurchaseorderNo" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Order no</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="contributionvoucherno" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="purchaseorderno" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="customersupplierno" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="customersuppliername" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>340</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="customersupplierlocation" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>340</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>BasarTableWidget</class>
   <extends>QTableWidget</extends>
   <header>libbasarqtwidget_tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rectificationcomment</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
